charts/opentelemetry-demo/examples/full/values.yaml

```diff
@@ -0,0 +1,182 @@
+observability:
```

Comment:
Are these values the same as the default values.yaml?  If so we can leave this file blank.